src/applications/vaos/tests/new-express-care-request/info-page.unit.spec.jsx

```diff
@@ -0,0 +1,126 @@
+import { expect } from 'chai';
```

Comment:
FYI, I copied this test over from the previously written unit test and tweaked it to use the layout component. There's not a whole lot to test here, so it seemed like consolidating into one file made the most sense.